Loading rhel8/default-icl
  Loading requirement: dot rhel8/slurm singularity/current rhel8/global
    cuda/11.4 intel/compilers/2020.2 intel/mkl/2020.2 intel/impi/2020.2/intel
    intel/libs/idb/2020.2 intel/libs/tbb/2020.2 intel/libs/ipp/2020.2
    intel/libs/daal/2020.2 intel/bundles/complib/2020.2
WARNING: Logging before flag parsing goes to stderr.
W0518 17:33:49.594148 23278654452608 lazy_loader.py:50] 
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

Warning: Flow failed to import. Set the environment variable D4RL_SUPPRESS_IMPORT_ERROR=1 to suppress this message.
No module named 'flow'
Warning: CARLA failed to import. Set the environment variable D4RL_SUPPRESS_IMPORT_ERROR=1 to suppress this message.
No module named 'carla'
2022-05-18 17:33:51,215	INFO node.py:498 -- Process STDOUT and STDERR is being redirected to /tmp/ray/session_2022-05-18_17-33-51_215399_273885/logs.
2022-05-18 17:33:51,325	INFO services.py:409 -- Waiting for redis server at 127.0.0.1:53471 to respond...
2022-05-18 17:33:51,437	INFO services.py:409 -- Waiting for redis server at 127.0.0.1:12774 to respond...
2022-05-18 17:33:51,438	INFO services.py:806 -- Starting Redis shard with 10.0 GB max memory.
2022-05-18 17:33:51,457	INFO node.py:512 -- Process STDOUT and STDERR is being redirected to /tmp/ray/session_2022-05-18_17-33-51_215399_273885/logs.
2022-05-18 17:33:51,457	WARNING services.py:1294 -- Warning: Capping object memory store to 20.0GB. To increase this further, specify `object_store_memory` when calling ray.init() or ray start.
2022-05-18 17:33:51,457	INFO services.py:1442 -- Starting the Plasma object store with 20.0 GB memory using /dev/shm.
2022-05-18 17:33:51,717	INFO tune.py:61 -- Tip: to resume incomplete experiments, pass resume='prompt' or resume=True to run()
2022-05-18 17:33:51,717	INFO tune.py:232 -- Starting a new experiment.
W0518 17:33:51.745438 23278654452608 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:131: The name tf.VERSION is deprecated. Please use tf.version.VERSION instead.

W0518 17:33:51.745675 23278654452608 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:136: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

2022-05-18 18:40:34,389	WARNING util.py:64 -- The `fetch_result` operation took 0.26635003089904785 seconds to complete, which may be a performance bottleneck.
W0518 18:40:34.448058 23278654452608 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:114: The name tf.Summary is deprecated. Please use tf.compat.v1.Summary instead.

2022-05-18 18:40:34,540	WARNING util.py:64 -- The `process_trial` operation took 0.4170873165130615 seconds to complete, which may be a performance bottleneck.
2022-05-18 18:46:47,426	WARNING util.py:64 -- The `save_to_disk` operation took 1.475773811340332 seconds to complete, which may be a performance bottleneck.
2022-05-18 18:46:47,435	WARNING util.py:64 -- The `process_trial` operation took 1.4915978908538818 seconds to complete, which may be a performance bottleneck.
2022-05-18 18:48:51,758	WARNING util.py:64 -- The `process_trial` operation took 0.10278701782226562 seconds to complete, which may be a performance bottleneck.
2022-05-18 18:49:33,327	WARNING util.py:64 -- The `experiment_checkpoint` operation took 0.11687445640563965 seconds to complete, which may be a performance bottleneck.
2022-05-18 18:53:41,312	WARNING util.py:64 -- The `save_to_disk` operation took 0.47574639320373535 seconds to complete, which may be a performance bottleneck.
2022-05-18 18:53:41,316	WARNING util.py:64 -- The `process_trial` operation took 0.48522019386291504 seconds to complete, which may be a performance bottleneck.
2022-05-18 19:00:32,015	WARNING util.py:64 -- The `save_to_disk` operation took 0.40839099884033203 seconds to complete, which may be a performance bottleneck.
2022-05-18 19:00:32,018	WARNING util.py:64 -- The `process_trial` operation took 0.4171466827392578 seconds to complete, which may be a performance bottleneck.
slurmstepd: error: *** JOB 61349449 ON cpu-q-139 CANCELLED AT 2022-05-18T19:04:00 DUE TO TIME LIMIT ***
